internal/transformers/typeeraser.go

```diff
@@ -9,10 +9,11 @@ import (
 type TypeEraserTransformer struct {
 	Transformer
 	compilerOptions *core.CompilerOptions
+	sourceFile      *ast.SourceFile
 }
 
-func NewTypeEraserTransformer(emitContext *printer.EmitContext, compilerOptions *core.CompilerOptions) *Transformer {
-	tx := &TypeEraserTransformer{compilerOptions: compilerOptions}
+func NewTypeEraserTransformer(emitContext *printer.EmitContext, sourceFile *ast.SourceFile, compilerOptions *core.CompilerOptions) *Transformer {
+	tx := &TypeEraserTransformer{compilerOptions: compilerOptions, sourceFile: sourceFile}
```

Comment:
This is normally picked up by capturing the source file in `visit` (or `visitSourceFile`) as it's likely that I will change transformers to be reused like they are in Strada (though changed to one per thread rather than one for all emit). See `runtimesyntax.go` or `commonjsmodule.go` for examples.